routes/index.go

```diff
@@ -70,10 +71,8 @@ func NewRouter() *http.Server {
 	r.Group(func(r chi.Router) {
 		r.Get("/lnauth_login", handlers.ReceiveLnAuthData)
 		r.Get("/lnauth", handlers.GetLnurlAuth)
-		r.Get("/lnauth_poll", handlers.PollLnurlAuth)
 		r.Get("/refresh_jwt", handlers.RefreshToken)
 		r.Post("/invoices", handlers.GenerateInvoice)
-		r.Get("/invoices/{payment_request}", handlers.GetInvoiceStatus)
```

Comment:
frontend is still requesting this endpoint, so its not working

Reply:
looks like this was an issue with me deploying without packr2 build, marking resolved